impy/definitions.py

```diff
@@ -104,9 +104,16 @@
     [
         'QGSJETII04', 'QGSJet', 'II-04', -1, 'qgsII04', qgsjet.QGSJetIIRun,
         qgsjet.QGSJETEvent, 'laboratory'
+    ],  
+    [
+        'SOPHIA20', 'SOPHIA', '2.0', -1, 'sophia', sophia.SophiaRun,
+        sophia.SophiaEvent, 'laboratory'
     ]
 ]
 
+# 'tag', 'name', 'version', 'crmc_id', 'library_name', 'RunClass',
+#     'EventClass', 'output_frame'
+
```

Comment:
Is this a leftover comment? Seems useful. Maybe, it's better to keep it above the block

Reply:
Yes, this is leftover from working with the code and it should be removed as it copy/paste from the code above